src/sentry/snuba/metrics/datasource.py

```diff
@@ -180,6 +181,7 @@ def get_custom_measurements(
     organization: Optional[Organization] = None,
     start: Optional[datetime] = None,
     end: Optional[datetime] = None,
+    use_case_id: UseCaseKey = UseCaseKey.RELEASE_HEALTH,
```

Comment:
default here should be performance as this is custom transaction measurements

Reply:
fixed.